catalogue/webapp/services/catalogue/works.js

```diff
@@ -45,7 +45,7 @@ export async function getWorks({
 }: GetWorksProps): Promise<CatalogueResultsList | CatalogueApiError> {
   const filterQueryString = Object.keys(filters).map(key => {
     const val = filters[key];
-    return `${key}=${val}`;
+    return val && `${key}=${val}`;
```

Comment:
Hmm. When is this not the case?

Reply:
if either the `_dateFrom` or `_dateTo` are null, the app will currently break without this check. But would it make more sense to conditionally add these to the filters only if they exist?

Reply:
I thought we dealt with this in the URL constructor?